tests/core/test_geometric_sources.py

```diff
@@ -161,9 +161,9 @@ def test_text3d_source():
 
 @pytest.mark.parametrize('string', [' ', 'TEXT'])
 @pytest.mark.parametrize('center', [(0, 0, 0), (1, -2, 3)])
-@pytest.mark.parametrize('height', [None, 0, 1, 2])
-@pytest.mark.parametrize('width', [None, 0, 1, 2])
-@pytest.mark.parametrize('depth', [None, 0, 1, 2])
+@pytest.mark.parametrize('height', [None, 0, 2.1])
+@pytest.mark.parametrize('width', [None, 0, 2.2])
+@pytest.mark.parametrize('depth', [None, 0, 2.3])
```

Comment:
```suggestion
# https://github.com/pyvista/pyvista/pull/6734
@pytest.mark.parametrize('height', [None, 0, 2.1])
@pytest.mark.parametrize('width', [None, 0, 2.2])
@pytest.mark.parametrize('depth', [None, 0, 2.3])
```

Reply:
I realized that comments are not necessary to make the redundancy appropriate. I will withdraw this proposal.